plugins/kafka-info/src/components/KafkaInfoComponent/KafkaInfoComponent.tsx

```diff
@@ -85,6 +108,24 @@ export function KafkaInfoComponent() {
     );
   }
 
+  const handleClusterChange = (event) => {
+    const selectedCluster = event.target.value;
+    setCurrentCluster(selectedCluster);
+  };
+
+  const ClusterSelect = () => (
+    <FormControl>
+      <InputLabel id="cluster-select-label">Cluster</InputLabel>
+      <Select labelId="cluster-select-label" id="cluster-select" value={currentCluster} onChange={handleClusterChange}>
+        {Object.keys(clusterMap).map((key) => (
```

Comment:
Because you are iterating over the clusters it might make sense to have an array of clusters rather than an object.

Reply:
Also, you could store the cluster object in state instead of an object name for looking up in the in the cluster object.

Reply:
is this how you meant?